doc/en/types/equality.md

```diff
@@ -63,6 +65,17 @@ Here, both operators compare for **identity** and **not** equality; that is, the
 will compare for the same **instance** of the object, much like `is` in Python 
 and pointer comparison in C.
 
+### Comparing NaN
```

Comment:
I'm not quite sure about this explanation, according to my testing and research *any* equality with NaN, double or triple, returns false.

I would suggest an alternate explanation:
```
Any equality comparison with `NaN` will return `false`. This includes comparing `NaN` with itself, making it the only value in javascript which doesn't equal itself.
```

I don't think it's relevant here to talk about the type of NaN, although that should probably be added in the `typeof` section.

Reply:
I've referred to this [ecma standards explanation](https://www.ecma-international.org/ecma-262/#sec-strict-equality-comparison)